app/controllers/admin/erip_transactions_controller.rb

```diff
@@ -117,9 +117,9 @@ def bepaid_notify
       p.payment_type = et.erip['service_no'].to_i == Setting['bePaid_serviceNo'].to_i ? 'membership' : 'donation'
       p.payment_form = 'erip'
       if p.payment_type == 'membership' then
-        last_payment = nil
-        last_payment = p.user.last_payment unless p.user.nil?
-        unless last_payment.nil? or last_payment.end_date < et.paid_at.to_date - 14.days
+        last_payment = p.user&.last_payment
```

Comment:
What does & do?

Reply:
obj&.method returns nil if obj is nil or calls method otherwise.

Reply:
https://ruby-doc.org/core-2.6/doc/syntax/calling_methods_rdoc.html#label-Safe+navigation+operator